lib/sms.rb

```diff
@@ -0,0 +1,21 @@
+require_relative "customer" # Not necessary except for the block below the class
+
+class Sms
+  attr_accessor :customer
+  attr_accessor :message
+
+  def initialize(customer)
+    @customer = customer
+    current_time = Time.now
+    deadline = current_time + 3600
+    @message = "Thank you! Your order was placed and will be delivered before #{deadline.strftime("%H:%M %p")}"
```

Comment:
Recommendation on best way to do that?  Just press `Enter` and go to a new line (then indent so looks nice)?